docs/instance_groups.md

```diff
@@ -320,3 +320,19 @@ spec:
     httpPutResponseHopLimit: 1
     httpTokens: required
 ```
+
+## maxInstanceLifetime (AWS Only)
+
+{{ kops_feature_table(kops_added_default='1.24') }}
```

Comment:
Given that we are (hopefully) days away from releasing 1.24, this probably will be 1.25.